publisher/server/publisher.go

```diff
@@ -115,6 +116,24 @@ func (s *Server) publishContainer(container *docker.APIContainers, ttl time.Dura
 			port := strconv.Itoa(int(p.PublicPort))
 			hostAndPort := s.host + ":" + port
 			if s.IsPublishableApp(containerName) && s.IsPortOpen(hostAndPort) {
+				configKey := fmt.Sprintf("/deis/config/%s/", appName)
+				// check if the user specified a healthcheck URL
+				healthcheckURL := s.getEtcd(configKey + "healthcheck_url")
+				delay, err := strconv.Atoi(s.getEtcd(configKey + "healthcheck_initial_delay"))
+				if err != nil {
+					log.Println(err)
```

Comment:
this will only show something like `strconv.ParseInt: parsing "blablabla": invalid syntax` in the logs. 
Why not showing this only in debug mode?